### PR TITLE
Upgrade user to admin

### PR DESCRIPTION
**Pull request by paradime**

### CODE REVIEW

**Comment by paradime** [comment]
image

File: src/routes/Group/components/admin/AdminView.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        <Header as="h4">Admins:</Header>
+            <Segment.Group>
+              {this.userUpgradeButtons('downgrade')}
+            </Segment.Group>
+      </Grid.Column>
+    </Grid>);
+  }
+
+  render() {
+    if (this.props.group.admins.includes(this.props.user._id)){
```
Comment (jarrydlee) [inline src/routes/Group/components/admin/AdminView.js]: nit: couple spacing and indentation fixes here

**Comment by paradime** [comment]
waiting on @kwm4385

File: package.json [hunk 2]
```diff
@@ -109,6 +109,7 @@
     "node-sass": "^3.7.0",
     "normalize.css": "^4.1.1",
     "postcss-loader": "^0.13.0",
+    "prop-types": "^15.5.8",
```
Comment (kwm4385) [inline package.json]: Why are you using this instead of the built in React proptypes?

File: src/store/group.js [hunk 3]
```diff
@@ -85,6 +87,34 @@ export const processRequest = (groupName, userId, approved) => {
   }
 };
 
+export const upgradeUser = (groupName, userId) => {
```
Comment (kwm4385) [inline src/store/group.js]: I'd change this name to be more consistent with the other, like `addAdmin`

File: package.json (hunk 2 above)
Comment (paradime) [inline package.json]: They deprecated them https://facebook.github.io/react/docs/typechecking-with-proptypes.html

**Pull request merged**